Lowest Common Ansestor.py

```diff
@@ -0,0 +1,75 @@
+class Node:
+    def __init__(self, info):
+        self.info = info
+        self.left = None
+        self.right = None
+        self.level = None
+
+    def __str__(self):
+        return str(self.info)
+
+
+class BinarySearchTree:
+    def __init__(self):
+        self.root = None
+
+    def create(self, val):
+        if self.root == None:
+            self.root = Node(val)
+        else:
+            current = self.root
+
+            while True:
+                if val < current.info:
+                    if current.left:
+                        current = current.left
+                    else:
+                        current.left = Node(val)
+                        break
+                elif val > current.info:
+                    if current.right:
+                        current = current.right
+                    else:
+                        current.right = Node(val)
+                        break
+                else:
+                    break
+
+
+# Enter your code here. Read input from STDIN. Print output to STDOUT
+'''
+class Node:
+      def __init__(self,info): 
+          self.info = info  
+          self.left = None  
+          self.right = None 
+
+
+       // this is a node of the tree , which contains info as data, left , right
+'''
+
+
+def lca(root, v1, v2):
+    # Enter your code here
+    current = root
+    while current:
+        if (current.info - v1) * (current.info - v2) <= 0:
+            return current
+        elif current.info > v1 and current.info > v2:

```

Comment:
`current.info > v2` is redundant. as elif implies ` (current.info - v1) * (current.info - v2) > 0`